### ISSUE TITLE
Remove use of `import *` in favor of specific module imports

### ISSUE THREAD

**Issue by matthewfeickert** [peterridolfi/Pyhf-to-Combine-converter#7]
Currently there is use of `import *` in

https://github.com/peterridolfi/Pyhf-to-Combine-converter/blob/2d154ad73ab460b8a6b7512f1cc8df5fb102ff0a/src/pyhf_convert_to_datacard.py#L13

and this should be avoided as it pollutes the local scope and undoes the benefit of a namespace.

```console
$ docker run \
    --rm \
    -ti \
    -P \
    --device /dev/fuse \
    --cap-add SYS_ADMIN \
    --security-opt apparmor:unconfined \
    -e CVMFS_MOUNTS="cms.cern.ch oasis.opensciencegrid.org" \
    pyhf/pyhf-combine-converter:cmssw-11.2.0-python3
MY_UID variable not specified, defaulting to cmsusr user id (1000)
MY_GID variable not specified, defaulting to cmsusr user group id (1000)
Mounting the filesystem "cms.cern.ch" ... DONE
Mounting the filesystem "oasis.opensciencegrid.org" ... DONE
Checking CVMFS mounts ... DONE
	The following CVMFS folders have been successfully mounted:
		cms.cern.ch
		oasis.opensciencegrid.org

# Sourcing cmsenv ... DONE
[cmsusr@ccab32cd61ab workarea]$ python
Python 2.7.15+ (default, Apr 16 2020, 00:57:30) 
[GCC 9.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from HiggsAnalysis.CombinedLimit.DatacardParser import * 
>>> dir()
['Datacard', 'FloatToString', 'FloatToStringScientific', '__builtins__', '__doc__', '__name__', '__package__', 'absolute_import', 'addDatacardParserOptions', 'addRateParam', 'doEditNuisance', 'fnmatch', 'globalNuisances', 'isIncluded', 'isVetoed', 'parseCard', 'print_function', 're', 'stderr', 'zip']
```

For libraries and tools it is important to only be importing the components of other libraries that you are actively using.

**Comment by peterridolfi** [comment]
Fixed

**Comment by matthewfeickert** [comment]
Resolved by d51a63612c072ea0c428a4e42b3c24f1eff31c1d
